docs/handlers.md

```diff
@@ -116,6 +116,6 @@ context, but it might also make sense to do other things, like cleanup after a t
 
 ## Handler errors
 
-(tbd)
+If an error happens within a Handler plugin, the Handler should handle the error internally.
```

Comment:
To asi chce nejake tbd, ne? Treba ty internal errory?

Reply:
jojo jen jsem to nechtel michat do jednoho PR (resp. ono uz to u Elastic a Slack handleru je, ale u Console a HTML to bude vetsi zasah a nechtel jsem tim tady delat noise, jakmile tohle merznu tak to udelam v dalsim PR ;-)